### ISSUE TITLE
Add itch.io page for testing the upload of this repo itself

### ISSUE THREAD

**Issue by janhohenheim** [TheBevyFlock/bevy-template#19]

**Comment by benfrankel** [comment]
This _may_ be incompatible with https://github.com/TheBevyFlock/bevy-template/issues/1.

**Comment by Sapein** [comment]
> This _may_ be incompatible with #1.

Assuming this would run in CI, we could just use cargo-generate to create the CI version that we're testing.

**Comment by benfrankel** [comment]
True, but then the CI would be wrong for the user?

There may be a way around this. IMO these are 2 important / useful features to support, so if a bit of a hack is needed to support both, it's worth it.

**Comment by benfrankel** [comment]
Unsure which user should create the itch.io page. There can be multiple admins, but only one owner, whose username also becomes part of the URL.

**Comment by janhohenheim** [comment]
I'd create a new user.

**Comment by benfrankel** [comment]
We now have https://the-bevy-flock.itch.io/bevy-template, so this issue is unblocked :)

**Comment by benfrankel** [comment]
Release workflow is functioning now, although the web release is broken because of `web/index.html`. Technically that's a separate issue, though: There is an itch.io page, and our broken game is uploaded to it automatically :tada:

**Comment by benfrankel** [comment]
Release workflow is functioning now, although the web release is broken because of `web/index.html`. Technically that's a separate issue, though: There is an itch.io page, and our broken game is uploaded to it automatically :tada:
